docs-markdown/src/test/suite/controllers/cleanup/removeEmptyMetadata.test.ts

```diff
@@ -0,0 +1,119 @@
+import * as chai from "chai";
+import * as spies from "chai-spies";
+import { resolve } from "path";
+import { commands, window, Uri } from "vscode";
+import { deleteEmptyMetadata, deleteNaMetadata, deleteCommentedMetadata } from "../../../../controllers/cleanup/removeEmptyMetadata";
+import * as removeEmptyMetadata from "../../../../controllers/cleanup/removeEmptyMetadata";
+import * as cleanup from "../../../../controllers/cleanup/cleanup-controller";
+import * as common from "../../../../helper/common";
+import * as telemetry from "../../../../helper/telemetry";
+
+chai.use(spies);
+
+const sinon = require("sinon");
+const expect = chai.expect;
+const testString = `---
+title: "Remove empty metadata markdown testing document" 
+ms.naValue: na
+ms.nsaValue: n/a
+#ms.commented: "comment"
+ms.list:
+    - "some value"
+    - "another value"
+ms.listWEmpty:
+    - ""
+    - "real value"
+    -
+ms.empty:
+ms.singleQuote: ''
+ms.doubleQuote: ""
+ms.realValue: "Hello World"
+---
+`
+
+suite("Remove Empty Metadata Controller", () => {
+    // Reset and tear down the spies
+    teardown(() => {
+        chai.spy.restore(common);
+    });
+    suiteTeardown(async () => {
+        await commands.executeCommand('workbench.action.closeAllEditors');
+    });
+    test("Remove Empty Metadata - Cleanup Controller", async () => {
```

Comment:
It appears that this test is not re-runable locally without undoing the changes from "removeEmptyMetadata". Please write or remove this so we can re-run the tests without failures.

Reply:
This is re-runnable in my environment, but I will investigate this issue further.